### PR TITLE
feat: api endpoints

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: lib/repository.ts [hunk 1]
```diff
@@ -2,54 +2,53 @@ import { ReadRequestOptions } from '@tigrisdata/core/dist/types'
...
-}
-
-export async function addItem (item: TodoItem) {
-
-}
-
-export async function updateItem (item: TodoItem) {
-
+export async function getItem (itemId: number): Promise<TodoItem | undefined> {
+  const db = tigris.getDatabase(DB_NAME)
```
Comment (ovaistariq) [inline lib/repository.ts:21]: Can the DB be stored as a global?

File: lib/repository.ts [hunk 2]
```diff
@@ -2,54 +2,53 @@ import { ReadRequestOptions } from '@tigrisdata/core/dist/types'
...
+  const collection: Collection<TodoItem> = db.getCollection(COLLECTION_NAME)
```
Comment (ovaistariq) [inline lib/repository.ts:22]: may be we could simplify it down to 1 liner

Comment (ovaistariq) [reply lib/repository.ts:22]: Once we make it global, this seems such minor code that we don't need a separate library and can have the call directly in the API function

**Comment by ovaistariq** [comment]
I have added comments, looks cool though. And later on, we could take it even further by auto-generating the API endpoints for the users as this is just boiler plate code.

File: lib/repository.ts (hunk 2 above)
Comment (adilansari) [reply lib/repository.ts]: Done

**Pull request merged**